app/compare/diff_chunk.tsx

```diff
@@ -24,38 +29,80 @@ export default class DiffChunk extends React.Component<DiffChunkProps, DiffChunk
     this.setState({ expanded: true });
   }
 
-  render() {
-    const {
-      change: { value, added, removed },
-    } = this.props;
-
-    const isUnchanged = !added && !removed;
-
-    const lines = value.trimEnd().split("\n");
-
-    const nodes = lines.map((line: string, index: number) => (
-      <div key={`${index}`} className={`diff-line ${added ? "added" : ""} ${removed ? "removed" : ""}`}>
-        <div className="plus-minus-cell">
-          {added && "+"}
-          {removed && "-"}
-        </div>
-        <div className="diff-line-content">{line}</div>
-      </div>
-    ));
-
-    if (nodes.length >= 10 && isUnchanged && !this.state.expanded) {
-      return [
-        nodes[0],
-        <OutlinedButton className="diff-line collapsed" onClick={this.onExpand.bind(this)}>
-          <div className="plus-minus-cell">
+  private renderAdded(text: string) {
+    return <ins className="added">{text}</ins>;
+  }
+
+  private renderRemoved(text: string) {
+    return (
+      <del className="removed">
+        <span>{text}</span>
```

Comment:
do we need a del and a span? can we just go with div or span?

Reply:
Span is needed for having the strikethrough be a different color than the text; see comments above re. accessibility